### PR TITLE
{system}[GCCcore/8.2.0] Hello v2.10

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
lg-1r17-n06 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/bb03674228be044fac73db0ed26bbec8 for a full test report.

**Comment by bartoldeman** [comment]
Hmm, perhaps, moduleclass should be tools?

File: easybuild/easyconfigs/h/Hello/Hello-2.10-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ('binutils', '2.31.1'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/hello'],
+    'dirs': ['share/man/man1'],
+}
+sanity_check_commands = ['hello']
+
+moduleclass = 'system'
```
Comment (boegel) [inline easybuild/easyconfigs/h/Hello/Hello-2.10-GCCcore-8.2.0.eb]: @bartoldeman `tools` makes more sense to me, indeed (certainly more than `system`)

I do wonder what your use case was here? :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2099.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f5415f394c64b0c211ee7dc3f81c1ee0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/641471d72b149f80921d2d2d713b1cc1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @bartoldeman!

**Pull request merged**